apriltags/build.gradle

```diff
@@ -0,0 +1,77 @@
+apply from: "${rootDir}/shared/resources.gradle"
+
+ext {
+    nativeName = 'apriltags'
+    devMain = 'edu.wpi.first.apriltags.DevMain'
```

Comment:
This and the top level folder use "apriltags" but the Java package is "apriltag".  We should be consistent here.  Also, we need to figure out how the thirdparty-apriltag bits work with this--are we going to add the JNI wrapper and library pull-ins to this same library?